### PR TITLE
Disable alpine official tests build

### PR DESCRIPTION
**Pull request by janvorli**

### CODE REVIEW

**Comment by janvorli** [comment]
@MattGal I did that according to your recommendation with one change though - instead of SkipTestBuild, I've used PB_**Enable**TestBuild to match the polarity of the PB_EnableCloudTest.

File: buildpipeline/DotNet-CoreFx-Trusted-Linux.json [hunk 1]
```diff
@@ -224,7 +225,7 @@
       "continueOnError": false,
       "alwaysRun": false,
       "displayName": "Create Helix Test Jobs",
-      "condition": "ne(variables['PB_EnableCloudTest'], 'false')",
+      "condition": "and(ne(variables['PB_EnableTestBuild'], 'false')), ne(variables['PB_EnableCloudTest'], 'false'))",
```
Comment (MattGal) [inline buildpipeline/DotNet-CoreFx-Trusted-Linux.json]: Incorrect # of parenthesis (one extra after first ne clause)

```
and(ne(variables['PB_EnableTestBuild'], 'false'), ne(variables['PB_EnableCloudTest'], 'false'))
```

Comment (janvorli) [reply buildpipeline/DotNet-CoreFx-Trusted-Linux.json]: Thanks for spotting that

**Comment by MattGal** [comment]
From discussions with @chcosta , and given that all other test builds are expected to be enabled by default currently, we both feel that naming this PB_DisableTestBuild makes more sense (I don't really care about skip vs. disable, more capturing that this is the exceptional behavior).  Otherwise, seems OK to me.

**Comment by janvorli** [comment]
Ok, let me change the naming and polarity.

**Comment by janvorli** [comment]
@MattGal done

**Comment by weshaggard** [comment]
@janvorli do we have a issue tracking the re-enabling of the alpine test runs?

FYI - I regressed this with my recent build changes to introduce PB_SkipTests option but I'm working to fix it.

**Comment by janvorli** [comment]
@weshaggard no, we don't, but we have an issue to enable the Alpine queue, which is necessary to enable the tests execution: https://github.com/dotnet/core-eng/issues/1799.
But as for the test building, we could re-enable that already, since the official core-setup packages are already produced and that's what was missing before to let us enable the test build.

**Pull request merged**